demo/uds/README.md

```diff
@@ -0,0 +1,13 @@
+This demo is used to show how to communicate with Layotto through UNIX.
```

Comment:
How about adding this doc to the official doc site and then [test it in our CI](https://mosn.io/layotto/#/zh/development/test-quickstart)

Reply:
> How about adding this doc to the official doc site and then [test it in our CI](https://mosn.io/layotto/#/zh/development/test-quickstart)

ok